### ISSUE TITLE
`zcashd --help` fails with an assertion error

### ISSUE THREAD

**Issue by superbaud** [zcash/zcash#6184]
```
sasha_z_cash@test-4gb-sasha:~/zcash$ ./src/zcashd --version
Zcash Daemon version v5.2.0-b2e4c0562

In order to ensure you are adequately protecting your privacy when using Zcash,
please see <https://z.cash/support/security/>.

Copyright (C) 2009-2022 The Bitcoin Core Developers
Copyright (C) 2015-2022 The Zcash Developers

This is experimental software.

Distributed under the MIT software license, see the accompanying file COPYING
or <https://www.opensource.org/licenses/mit-license.php>.
sasha_z_cash@test-4gb-sasha:~/zcash$ ./src/zcashd --help
zcashd: chainparamsbase.cpp:86: const CBaseChainParams &BaseParams(): Assertion `pCurrentBaseParams' failed.
Aborted
sasha_z_cash@test-4gb-sasha:~/zcash$ 
```

**Comment by daira** [comment]
I can reproduce this. I'll try to bisect what caused it tomorrow.

**Comment by superbaud** [comment]
Fixed by #6204
